db/migrate/20200814153802_add_all_nation_applicability_to_edition.rb

```diff
@@ -0,0 +1,5 @@
+class AddAllNationApplicabilityToEdition < ActiveRecord::Migration[5.1]
+  def change
+    add_column :editions, :all_nation_applicability, :boolean
```

Comment:
Can this have a `default: false`? (Or are we likely to see validation errors with older editions?)

Reply:
Whoops! 

I think that should be OK provided the data migration runs and sets any older editions without any inapplicabilities to `all_nation_applicability: true`.

Perhaps the default should be `true` though as that is the current behaviour unless the user opts out, and also for the other Edition types without the concept of `nation_inapplicability` that will now have this (unused) attribute.

Reply:
Hmm yes, I think `default: true` does make more sense! Do we know how long the data migration runs for?

Reply:
Locally it took about 7 minutes with 156,849 records.
Integration has 165,733 and staging 165,777.

Reply:
I guess this means during that time, there's the potential for validation errors on the existing editions that haven't been updated yet (although users can fix it manually by ticking or unticking the box?)?.

Reply:
I think that is correct - any validations could be user corrected. 

I also think this is going to be a managed release - there will be some comms around it so we could possibly ask users not to update for a half-hour window.

Reply:
Cool, that makes sense 👍

Reply:
i can also now remove updating the record to `true` in the migration to save time.

Reply:
it seems that an unchecked check box will not send a "false" value, but merely nothing...meaning that an unchecked box will now default to true.